src/main/java/ch/njol/skript/command/Commands.java

```diff
@@ -241,10 +240,15 @@ static boolean handleEffectCommand(final CommandSender sender, String command) {
 	public static ScriptCommand getScriptCommand(String key) {
 		return commands.get(key);
 	}
-	
+
 	public static boolean skriptCommandExists(final String command) {
 		final ScriptCommand c = commands.get(command);
-		return c != null && c.getName().equals(command);
+		return c != null && (c.getName().equals(command) || c.getAliases().contains(command));
+	}
+
+	public static boolean commandExists(final String command) {
+		final Command c = getCommandMap().getCommand(command);
+		return c != null && (c.getName().equals(command) || c.getAliases().contains(command));
```

Comment:
Does the aliases check apply for Bukkit commands? You might need to reference the aliases map (I think that's a thing) instead

Reply:
Hey sorry for late response taking time to complete this and push out changes but it does seem to work yes as is with no changes
![image](https://user-images.githubusercontent.com/81952476/210125935-efd4db23-70d3-4198-9aab-54e962d070fc.png)